waspc/cli/Command/Call.hs

```diff
@@ -9,4 +9,5 @@ data Call
   | Build
   | Version
   | Telemetry
+  | Commands [String]
```

Comment:
Since this can really be a one String or no Strings, let's make it `Commands (Maybe String)`?